AzFileDiagnostics/Windows/AzFileDiagnostics.ps1

```diff
@@ -375,6 +375,45 @@ function ValidateLmCompatibilityLevel {
 
 
     
+}
+
+##############################
+#.SYNOPSIS
+# Validate the RestrictSendingNTLMTraffic. 
+#
+#.DESCRIPTION
+# Currently just check the registry key and its value. if it is set to 0 or 1, prompt user to change it to expected value. 
```

Comment:
Should be "if it is not set to 0 or 1, prompt user to change it to expected value.

Reply:
Add some context on what the setting means to users.